atlasdb-autobatch/src/main/java/com/palantir/atlasdb/autobatch/DisruptorAutobatcher.java

```diff
@@ -81,7 +88,12 @@ private static ThreadFactory createThreadFactory(String safeLoggablePurpose) {
     @Override
     public void close() {
         closed = true;
-        disruptor.shutdown();
+        try {
+            disruptor.shutdown(10, TimeUnit.SECONDS);
```

Comment:
Not sure it even makes sense to do the shutdown then, but this is fine since we can see some logging if we are hitting it at least